public/index.html

```diff
@@ -3377,20 +3405,20 @@ <h4>Chat/Message Handling</h4>
                                         "Send" to Continue
                                     </span>
                                 </label>
-                                <label class="checkbox_label" for="quick_continue">
+                                <label class="checkbox_label" for="quick_continue" title="Show a button in the input area to ask the AI to continue (extend) its last message." data-i18n="[title]Show a button in the input area to ask the AI to continue (extend) its last message.">
                                     <input id="quick_continue" type="checkbox" />
                                     <span data-i18n="Press Send to continue">
                                         Quick "Continue" button
                                     </span>
                                 </label>
                                 <div class="checkbox-container flex-container">
-                                    <label data-newbie-hidden class="checkbox_label" for="swipes-checkbox">
+                                    <label data-newbie-hidden class="checkbox_label" for="swipes-checkbox" title="Allow generating alternative responses from the AI before choosing which one to commit. Shows arrow buttons next to the AI's last message. Both PC and mobile." data-i18n="[title]Allow generating alternative responses from the AI before choosing which one to commit. Shows arrow buttons next to the AI's last message. Both PC and mobile.">
```

Comment:
Difficult to explain compactly. Tried again, see the new commit.

Also, since the *Swipes* and *Gestures* checkboxes are on the same row in the UI - which I think is the right choice, since they are related and each label is just one word - for clarity, I added both desktop and mobile icons for *Swipes*, so it's clear that it applies to both although *Gestures* applies only to mobile.

Reply:
Actually, why not ask the LLM... Dolphin-Mistral came up with this:

*"Click arrows for alt. AI responses, choose your favorite, rest automatically disappear."*

I could change it to a variant of this if that's preferable?

Reply:
That's how I've understood it works. Aren't they?

Reply:
And yes, it's an easy point of confusion, because many LLM power users have likely read about [loom](https://generative.ink/posts/loom-interface-to-the-multiverse/) and at first glance, swipes sounds a lot like that.

I'd actually love to have some loom-like functionality for creative writing. I've noticed that ST has something called "checkpoints" and "branches", but I couldn't find any docs on that, and haven't had the time to do the detective work yet.

Reply:
Aah, I hadn't noticed that. Makes it even more complex to explain compactly.

I'll see what I can come up with.

Reply:
Ended up keeping the current simple tooltip for now:

*"Show arrow buttons on the last in-chat message to generate alternative AI responses. Both PC and mobile."*